tests/acceptance/01_vars/02_functions/datatype.cf.expected.json

```diff
@@ -0,0 +1,123 @@
+{
+  "container": {
+    "array": [
+      1,
+      2,
+      3
+    ],
+    "false": false,
+    "int": 100,
+    "null": null,
+    "real": 100.2300,
+    "string": "",
+    "true": true
+  },
+  "datatype[container[array]]": "data array",
+  "datatype[container[false]]": "data boolean",
+  "datatype[container[int]]": "data number",
+  "datatype[container[null]]": "data null",
+  "datatype[container[real]]": "data number",
+  "datatype[container[string]]": "data string",
+  "datatype[container[true]]": "data boolean",
+  "datatype[container]": "data object",
+  "datatype[i]": "primitive int",
+  "datatype[il]": "list ilist",
+  "datatype[init.container[array]]": "data array",
+  "datatype[init.container[false]]": "data boolean",
+  "datatype[init.container[int]]": "data number",
+  "datatype[init.container[null]]": "data null",
+  "datatype[init.container[real]]": "data number",
+  "datatype[init.container[string]]": "data string",
+  "datatype[init.container[true]]": "data boolean",
+  "datatype[init.container]": "data object",
```

Comment:
"primitive string" or "list slist" or whatever was used originally, because arrays are just a regular variable with brackets in the name.